src/Infrastructure/Helper/TextCaseHelper.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Utils\Infrastructure\Helper;
+
+use Laminas\Filter\FilterChain;
+use Laminas\Filter\StringToLower;
+use Laminas\Filter\Word\CamelCaseToDash;
+use Laminas\Filter\Word\DashToCamelCase;
+use SprykerSdk\Utils\Infrastructure\Helper\Filter\CamelCaseToDash as CamelCaseToDashWithoutAbbreviation;
+
+class TextCaseHelper
+{
+    /**
+     * @param string $value
+     * @param bool $separateAbbreviation
+     *
+     * @return string
+     */
+    public static function camelCaseToDash(string $value, bool $separateAbbreviation = true): string
+    {
+        $filterChain = new FilterChain();
```

Comment:
Why do we need to use laminas when we can use a lightweight raw PHP functions?
Example -> https://github.com/illuminate/support/blob/master/Str.php#L1310-L1325

Reply:
I have extracted logic from the upgrader and do not expect to totally rewrite it

Reply:
Discussed